api/specs/webserver/openapi-viewer.yaml

```diff
@@ -0,0 +1,90 @@
+components:
+  schemas:
+    FielType2ViewerEnveloped:
+      properties:
+        data:
+          $ref: './openapi-viewer.yaml#/components/schemas/FileType2Viewer'
+      required:
+      - data
+      title: FielType2ViewerEnveloped
+      type: object
+    FieldType2ViewerListEnveloped:
+      properties:
+        data:
+          items:
+            $ref: './openapi-viewer.yaml#/components/schemas/FileType2Viewer'
+          type: array
+      required:
+      - data
+      title: FieldType2ViewerListEnveloped
+      type: object
+    FileType2Viewer:
+      properties:
+        file_type:
+          title: File Type
+          type: string
+        redirection_url:
+          description: Base url to redirect to this viewer. Needs appending file_size,
+            [file_name] and download_link
+          format: uri
+          maxLength: 2083
+          minLength: 1
+          title: Redirection Url
+          type: string
+        viewer_title:
+          description: Short formatted label with name and version of the viewer
+          title: Viewer Title
+          type: string
+      required:
+      - file_type
+      - viewer_title
+      - redirection_url
+      title: FileType2Viewer
+      type: object
+paths:
+  /viewers:
+    get:
+      operationId: get_viewer_for_file
+      parameters:
+      - in: query
+        name: file_type
+        required: true
+        schema:
+          title: File Type
+          type: string
+      - in: query
+        name: file_name
+        required: false
+        schema:
+          title: File Name
+          type: string
+      - in: query
+        name: file_size
+        required: false
+        schema:
```

Comment:
MINOR and as a note for future: I think all these openAPI variables should normally be CamelCase. Then in the Pydantic models you can issue ```BaseModel.dict(by_alias=True)``` when generating payloads.
The frontend prefers camelcasing... If we don't do it, then at least maybe we should define it cause currently we have a mixture.

Reply:
Good point. So far we have a mixture. I add that to the list of things to do for the web API